packages/workspace/src/utilities/typescript/compilation.ts

```diff
@@ -145,7 +145,7 @@ function getNormalizedTsConfig(options: TypeScriptCompilationOptions) {
   const tsConfig = readTsConfig(options.tsConfig);
   tsConfig.options.outDir = options.outputPath;
   tsConfig.options.noEmitOnError = true;
-  tsConfig.options.rootDir = options.rootDir;
+  tsConfig.options.rootDir = '.';
```

Comment:
the fix is based on conversation from here: https://github.com/nrwl/nx/issues/11583
The only problem is that `options.root` (as suggested in comments) are always `undefined`. Tested and 
```
tsConfig.options.rootDir = undefined;
```
Works as well. Decided to use `'.'` instead of `undefined` as similar approach is used in several other places in codebase